frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,6 +2,9 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
+  banner-simple(severity='warning' class='c-banner' v-if='!hasIncomeDetails && isCloseToDistributionTime')
```

Comment:
I know this hasn't been added to the designs yet, but it's been requested on Figma and am just waiting to hear back from @leihla. Do you think it would be possible to add an 'x' to the right-hand side to dismiss the banner manually?

Reply:
Oh - my bad @leihla! I didn't see your update, the X is there on Figma, thank you! 🎉